### DISCUSSION TITLE
How to handle open EventGroups (Finances API)

### DISCUSSION THREAD

**Discussion by CrackHead007** [saleweaver/python-amazon-sp-api#530] [Q&A]
Hi,

Anyone got any ideas how to handle open EventGroups that have already been saved to a DB? How can I update events that were added after I saved a specific group? Do I have to do it manually or is there a way to implement it within the API.

Thanks

**Comment by zyzjian** [comment]
hello   i am using Eventgroups to get my datas, but I get the same data when I use the same group ID and a different Next Tken,  Have you ever been in the same situation? Looking forward to your help
